_posts/blog/2019-06-23-About-Georgia.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: About Georgia
+categories: blog
+excerpt: Researcher in Kirstie's lab working on citizen science
+tags: [onboarding, introducing]
+image:
+  feature:
+link:
+date: 2019-06-14
+modified:
+share: true
+author: GeorgiaHCA
+---
+
+## What I do at the Turing
+
+I joined the Turing Institute for a 2-year contract, based in Kirstie’s lab, working on participatory science, sensory processing, and autism. My research interests include data ethics and the philosophy of technology. I’m currently building a citizen science platform and developing a participatory science framework.
+
+Before joining the Turing I was a civil servant, and I studied English literature for my undergrad and M.Phil degrees.
+
+I am open to collaborating and am interested in helping out with projects and papers – so get in touch if you’d like to join forces!
+
+## Where I'm From
+
+<figure>
+  <img src="/images/Lauder.jpg"
```

Comment:
I think this image is missing.

Also, please can you make a sub folder for the images and put them in there? Otherwise the `images/` folder is going to get really cluttered....

You don't have to move your profile picture btw (in fact, you shouldn't! It will all break if you do!)